spec/unit/cli_spec.rb

```diff
@@ -54,7 +54,7 @@ def stderr
     E
   end
 
-  let(:version_message) { "Chef Development Kit Version: #{ChefDK::VERSION}\n" }
+  let(:version_message) { "Chef Development Kit version: #{ChefDK::VERSION}\n" }
```

Comment:
What do you think about the specs using the strings from the Dist constants in their expectations? I think given the current state, the test suite won't pass without direct edits to the specs themselves when Dist is overridden.

Reply:
I was sorta considering that out of scope for this bit. I know the community folks are going to need that when they jump into cutting their own releases, but they'll need to get the specs cleaned up.

Reply:
Sure. Could be a Known Thing To Do™ for the first effort to distribute WorkDK as something else.